assets/css/style.css

```
@@ -109,18 +210,16 @@ button {
   margin: 0 auto;
   max-width: 970px;
   text-align: center;
-  margin-top: 130px;
   text-align: center;
 }
 
 .title-decoration {
   display: block;
-  margin: 0 auto; 
+  margin: 0 auto;
   text-align: center;
 }
 
 .wrapper {
   max-width: 1600px;
   margin: 0 auto;
-}
-
+}
```

```suggestion
}

```
always add blank line in the end of the file